### ISSUE TITLE
PactDirectory Error After Consumer Test Passes

### ISSUE THREAD

**Issue by dckallos** [pact-foundation/pact-jvm#1350]
My test is referencing a `PactDirectory` that is undefined, and it is throwing this error: 

```
java.lang.NoClassDefFoundError: au/com/dius/pact/core/model/annotations/PactDirectory

	at au.com.dius.pact.consumer.junit5.PactConsumerTestExt.lookupPactDirectory(PactConsumerTestExt.kt:397)
	at au.com.dius.pact.consumer.junit5.PactConsumerTestExt.afterAll(PactConsumerTestExt.kt:409)
```

This happens after the test passes, so there is some issue in creating the contract to the specified directory. When does this error arise, and how is it resolved? I'm unable to find `PactDirectory` anywhere in the code base. 

Build tool: Maven 3.6.3
Extension: `pact-jvm-consumer-junit5`
Version: `4.1.18`

**Comment by dckallos** [comment]
The issue is resolved by upgrading to `4.2.4`. The `@PactDirectory` annotation appears as an option, and `mvn test` succeeds. 

The `4.1.x` build should have made `@PactDirectory` available as well, so I'm not sure what happened there.

**Comment by uglyog** [comment]
`@PactDirectory` was added to 4.2.0 as a replacement of `@PactFolder` which is now deprecated.
